.github/workflows/yocto-build-deploy.yml

```diff
@@ -0,0 +1,639 @@
+name: "Yocto"
+
+on:
+  workflow_call:
+    secrets:
+      BALENA_API_KEY:
+        description: balena API key for deploying releases
+        required: false
+      S3_ACCESS_KEY:
+        description: S3 access key
+        required: false
+      S3_SECRET_KEY:
+        description: S3 secret key
+        required: false
+      DOCKERHUB_USER:
+        description: Dockerhub user for pulling private helper images
+        required: false
+      DOCKERHUB_TOKEN:
+        description: Dockerhub token for pulling private helper images
+        required: false
+      SIGN_KMOD_KEY_APPEND:
+        description: Base64-encoded public key of a kernel module signing keypair
+        required: false
+      SIGN_API_KEY:
```

Comment:
The description is wrong here. This is the balenaCloud API key that gives access to the signing server. Nothing to do with module signing, those only need to include the public key above.

Reply:
ack, I'll update it